### PR TITLE
Simplified example for Variables in JS

### LINKED ISSUE #6226: Issue with "Storing the information you need — Variables": Poor Counter Example for the Importance of Variables
MDN URL: https://developer.mozilla.org/en-US/docs/Learn/JavaScript/First_steps/Variables

#### What information was incorrect, unhelpful, or incomplete?
In the Section "What is a Variable?", the counter example meant to show why a world without variables would be bad uses the same variable as "Good" case, but just in some strange contrived way. This particular example wouldn't even suffer much from a world without variables.

  alert('Hello ' + prompt('What is your name?'); + ', nice to see you!');

A better example would be to use some other code snippet to show the need for the repeated retrieval of the same data from the user over and over again.

#### Specific section or headline?
"What is a Variable?"

#### What did you expect to see?
An example that didn't use variables when trying to prove that variables were important.

#### Did you test this? If so, how?
Yes. I ran locally.

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/learn/javascript/first_steps/variables`
* MDN URL: https://developer.mozilla.org/en-US/docs/Learn/JavaScript/First_steps/Variables
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/learn/javascript/first_steps/variables/index.html
* Last commit: https://github.com/mdn/content/commit/be3c3e06d1d9b59d2ad38a3a3e9eb59fbaa9c6f1
* Document last modified: 2021-06-10T18:50:18.000Z

</details>

### PR DESCRIPTION
**Pull request by FarazzShaikh**
> What was wrong/why is this fix needed? (quick summary only)

The example for why variables are useful [here](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/First_steps/Variables) was overly complicated.

> Issue number (if there is an associated issue)

Fixes #6226 

> Anything else that could help us review it

### CODE REVIEW

File: files/en-us/learn/javascript/first_steps/variables/index.html [hunk 1]
```diff
@@ -38,46 +38,60 @@ <h2 id="Tools_you_need">Tools you need</h2>
...
-button.onclick = function() {
-  let name = prompt('What is your name?');
-  alert('Hello ' + name + ', nice to see you!');
-}</pre>
+<p>To understand why this is so useful, let's think about how we'd write this example without using a variable. It would end up looking something like this:</p>
 
-<p>{{ EmbedLiveSample('What_is_a_variable', '100%', 50, "", "", "hide-codepen-jsfiddle") }}</p>
 
-<p>In this example pressing the button runs a couple of lines of code. The first line pops a box up on the screen that asks the reader to enter their name, and then stores the value in a variable. The second line displays a welcome message that includes their name, taken from the variable value.</p>
+<div id="example_without_variabele">
```
Comment (wbamberg) [inline files/en-us/learn/javascript/first_steps/variables/index.html:70]:
```suggestion
<div id="example_without_variable">
```

File: files/en-us/learn/javascript/first_steps/variables/index.html [hunk 2]
```diff
@@ -38,46 +38,60 @@ <h2 id="Tools_you_need">Tools you need</h2>
...
+    <pre class="brush: js example-bad">
+    const buttonB = document.querySelector('#button_B');
+    const headingB = document.querySelector('#heading_B');
+    
+    buttonB.onclick = function() {
+        alert('Hello ' + prompt('What is your name?') + ', nice to see you!');
+        headingB.textContent = 'Welcome ' + prompt('What is your name?');
+    }</pre>
+    
+    <p>{{ EmbedLiveSample('example_without_variabele', '100%', 50, "", "", "hide-codepen-jsfiddle") }}</p>
```
Comment (wbamberg) [inline files/en-us/learn/javascript/first_steps/variables/index.html:86]:
```suggestion
    <p>{{ EmbedLiveSample('example_without_variable', '100%', 50, "", "", "hide-codepen-jsfiddle") }}</p>
```

File: files/en-us/learn/javascript/first_steps/variables/index.html [hunk 3]
```diff
@@ -38,46 +38,60 @@ <h2 id="Tools_you_need">Tools you need</h2>
...
+</div>
 
-// ... and so on ...</pre>
 
-<p>You may not fully understand the syntax we are using (yet!), but you should be able to get the idea — if we didn't have variables available, we'd have to implement a giant code block that checked what the entered name was, and then display the appropriate message for any name. This is obviously really inefficient (the code is a lot bigger, even for only five choices), and it just wouldn't work — you couldn't possibly store all possible choices.</p>
+<p>You may not fully understand the syntax we are using (yet!), but you should be able to get the idea — if we didn't have variables available, we'd have to ask the reader for their name multiple times! Not only does this negitively affect user experience, this is obviously really inefficient (the code is longer), and it just wouldn't work — you couldn't possibly ask the reader their name every time you need it.</p>
```
Comment (wbamberg) [inline files/en-us/learn/javascript/first_steps/variables/index.html:90]:
```suggestion
<p>You may not fully understand the syntax we are using (yet!), but you should be able to get the idea. If we didn't have variables available, we'd have to ask the reader for their name every time we needed to use it!</p>
```

**Review by wbamberg — CHANGES_REQUESTED** [review]
Thanks for this PR, it's a much, much better example of why we need variables!

I fixed a typo and had a suggestion for rewording and simplifying the text.

**Comment by FarazzShaikh** [comment]
Thank you! Sounds good, I have made the changes

**Review by wbamberg — APPROVED** [review]
👍 thanks for your contribution!

**Pull request merged**